### PR TITLE
Make grpc_cli more useful

### PR DESCRIPTION
**Pull request by Cloudef**
Fixes issue https://github.com/grpc/grpc/issues/11290
Makes it actually possible to deal with stream responses.

I'm sort of sad with the state of GRPC tools, documentations (especially on the wire level) and the quality of implementation. Doesn't really make me feel like it's mature or ready for anything at all.

### CODE REVIEW

**Comment by grpc-testing** [comment]
Thanks for your pull request. The automated tests will run as soon as one of the admins verifies this change is ok for us to run on our infrastructure.

**Comment by grpc-testing** [comment]
Thanks for your pull request. The automated tests will run as soon as one of the admins verifies this change is ok for us to run on our infrastructure.

**Comment by grpc-testing** [comment]
Thanks for your pull request. The automated tests will run as soon as one of the admins verifies this change is ok for us to run on our infrastructure.

**Comment by grpc-testing** [comment]
Thanks for your pull request. The automated tests will run as soon as one of the admins verifies this change is ok for us to run on our infrastructure.

**Comment by grpc-testing** [comment]
Thanks for your pull request. The automated tests will run as soon as one of the admins verifies this change is ok for us to run on our infrastructure.

**Comment by Cloudef** [comment]
I signed it!

**Comment by yang-g** [comment]
jenkins: this is ok to test

**Comment by y-zeng** [comment]
Thanks for the pull request! Please format the change as clang-format suggested [here](https://grpc-testing.appspot.com/job/gRPC_pull_requests_linux/5657/testReport/junit/(root)/sanity_linux_dbg_native/tools_distrib_clang_format_code_sh/).

File: test/cpp/util/grpc_tool.cc [hunk 1]
```diff
@@ -72,6 +75,7 @@ DEFINE_string(proto_path, ".", "Path to look for the proto file.");
 DEFINE_string(protofiles, "", "Name of the proto file.");
 DEFINE_bool(binary_input, false, "Input in binary format");
 DEFINE_bool(binary_output, false, "Output in binary format");
+DEFINE_bool(repeated, false, "Wrap binary messages into parent protobuf message with tag number 1");
```
Comment (y-zeng) [inline test/cpp/util/grpc_tool.cc]: Please format the change as clang-format suggested [here](https://grpc-testing.appspot.com/job/gRPC_pull_requests_linux/5657/testReport/junit/(root)/sanity_linux_dbg_native/tools_distrib_clang_format_code_sh/).

**Comment by Cloudef** [comment]
I did the changes btw

**Comment by y-zeng** [comment]
Jenkins: test this please

**Comment by grpc-testing** [comment]
Thanks for your pull request. The automated tests will run as soon as one of the admins verifies this change is ok for us to run on our infrastructure.

**Comment by grpc-testing** [comment]
Thanks for your pull request. The automated tests will run as soon as one of the admins verifies this change is ok for us to run on our infrastructure.

**Comment by y-zeng** [comment]
Jenkins: test this please

**Comment by grpc-testing** [comment]
Thanks for your pull request. The automated tests will run as soon as one of the admins verifies this change is ok for us to run on our infrastructure.

**Comment by y-zeng** [comment]
Known issues:
Basic Tests Linux (opt): #11737
Basic Tests Mac: #11116

**Comment by thelinuxfoundation** [comment]
Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.

After the following items are addressed, please respond with a new comment here, and the automated system will re-verify.

* User @Cloudef isn't covered by a CLA. They will need to complete the form at https://identity.linuxfoundation.org/projects/cncf

_Regards,
CLA GitHub bot_

**Pull request closed without merging**